auth/api/iam/api.go

```diff
@@ -255,6 +243,22 @@ func (r Wrapper) IntrospectAccessToken(_ context.Context, request IntrospectAcce
 		log.Logger().WithError(err).Error("IntrospectAccessToken: failed to marshal presentation submission")
 		return IntrospectAccessToken200JSONResponse{}, err
 	}
+
+	if token.InputDescriptorConstraintIdMap != nil {
+		// Make sure InputDescriptorConstraintIdMap does not override existing properties (e.g. client_id)
+		// We could use a hardcoded list, but that risks getting outdated. So just remarshal into a map
+		// and check if any of the fields would be overwritten.
```

Comment:
(nearly) all values are optional meaning that they could (somewhere in the future) be missing from the marshaled response map. An omitted claim could then be filled by a IDC_ID. Probably an acceptable edge case?